tracer/test/Datadog.Trace.Tests/SpanTests.cs

```diff
@@ -160,5 +160,114 @@ public void TopLevelSpans()
                 scope.Span.Should().Match<Span>(s => s.IsTopLevel == expectedResult);
             }
         }
+
+        [Fact]
+        public void SpanIds()
+        {
```

Comment:
My preference would be to break out each scope into its own unit test to ensure that the `_tracer` accesses for each test are independent of each other, since each test case will initiate the constructor

Reply:
Good point. Done.

✔️